examples/editable-layers/codesandbox/getting-started/package.json

```diff
@@ -2,12 +2,11 @@
   "dependencies": {
     "@deck.gl/core": "^8.8.23",
     "@deck.gl/react": "^8.8.23",
-    "@deck.gl-community/react": "0.0.1",
```

Comment:
For the time being I'm omitting unpublished packages from `package.json`, since it causes `yarn add x` to fail on these examples, and confuses ESLint.